shared/src/main/scala/com/raquo/domtypes/generic/defs/eventProps/ClipboardEventProps.scala

```diff
@@ -7,20 +7,20 @@ import com.raquo.domtypes.generic.builders.EventPropBuilder
   *
   * For type param docs see [[EventPropBuilder]]
   */
-trait ClipboardEventProps[EP[_ <: DomEvent], DomEvent] { this: EventPropBuilder[EP, DomEvent] =>
+trait ClipboardEventProps[EP[_ <: DomEvent], DomEvent, ClipboardEvent <: DomEvent] { this: EventPropBuilder[EP, DomEvent] =>
```

Comment:
Pls rename to `DomClipboardEvent` to follow the naming convention (DOM type params start with "Dom")